spec/integration/rails_spec.rb

```diff
@@ -12,41 +12,31 @@
   require 'action_controller/railtie'
   require 'action_mailer/railtie'
 
-  # We don't use :mock_intake, as we want the stubs to stay around between
-  # individual examples
-  RSpec.describe 'Rails integration', :allow_running_agent, :spec_logger do
+  RSpec.describe 'Rails integration' do
     include Rack::Test::Methods
-    include MockIntake::WaitFor
+    include_context 'event_collector'
```

Comment:
I don't have any objections but are there reasons you prefer `shared_context` over the example tags approach that's used elsewhere? (Eg, `describe "thing", :mock_intake`)

Reply:
I think including the context is more explicit. When you see a `include_context`, you assume that there are some `before`/`after`/`let`/etc blocks defined + included. With the `:mock_intake` metadata, it's not clear to me whether it does something like what `:allow_running_agent` does (raise an exception if some condition is not met) or if it defines `before`/`after`/`let`/etc blocks that have a direct role in the test. In this case, what we want to include, is, in fact, an extra context so I think it makes most sense to define it as such.